tests/bson/bson-utcdatetime-clone-001.phpt

```diff
@@ -0,0 +1,30 @@
+--TEST--
+MongoDB\BSON\UTCDateTime #001
+--INI--
+date.timezone=America/Los_Angeles
```

Comment:
Is this necessary?

I realize several other tests include a `date.timezone` INI setting even though they only deal with the milliseconds and never interact with DateTime conversion. It may be redundant in these files as well:

 * bson-fromPHP-002.phpt (this doesn't even use UTCDateTime at all)
 * bson-fromPHP-003.phpt
 * bson-fromPHP_error-003.phpt
 * bson-utcdatetime-001.phpt
 * bson-utcdatetime-tostring-001.phpt

Reply:
It's not necessary in this case, so I removed it. I've created [PHPC-1470](https://jira.mongodb.org/browse/PHPC-1470) for the rest of them.